hedera-node/src/main/java/com/hedera/services/state/merkle/MerkleOptionalBlob.java

```diff
@@ -28,14 +28,19 @@
 import com.swirlds.common.io.SerializableDataOutputStream;
 import com.swirlds.common.merkle.MerkleExternalLeaf;
 import com.swirlds.common.merkle.utility.AbstractMerkleLeaf;
+import com.swirlds.common.merkle.utility.Keyed;
 
 import java.io.IOException;
 import java.util.Objects;
 import java.util.function.Supplier;
 
-public class MerkleOptionalBlob extends AbstractMerkleLeaf implements MerkleExternalLeaf {
-	static final int MERKLE_VERSION = BinaryObject.ClassVersion.ORIGINAL;
+public class MerkleOptionalBlob extends AbstractMerkleLeaf implements MerkleExternalLeaf, Keyed<String> {
+	static final int PRE_RELEASE_0180_VERSION = 1;
+	static final int RELEASE_0180_VERSION = 2;
+
+	static final int CURRENT_VERSION = RELEASE_0180_VERSION;
```

Comment:
nit: Is `MERKEL_VERSION` semantically clearer than `CURRENT_VERSION`?

Reply:
I'm liking `CURRENT_VERSION` more these days...what do you think?

Reply:
Yeah, it makes sense after going through the usages.